modules/p2/ding_list/theme/theme.inc

```diff
@@ -164,24 +165,28 @@ function theme_ding_list_lists($variables) {
       }
     }
 
+    if (!empty($rows)) {
+      $out .= theme('item_list', array(
```

Comment:
Avoid calling theme() directly. It is preferable to replace direct calls to the theme() function with calls to drupal_render() by passing a render array with a #theme key to drupal_render(), which in turn calls theme().
See: https://api.drupal.org/api/drupal/includes!theme.inc/function/theme/7.x